src/sd-jwt/index.ts

```diff
@@ -36,6 +36,7 @@ export const decode = <S extends z.AnyZodObject>(
     header: decodedJwt.protectedHeader,
     payload: decodedJwt.payload,
   });
+  console.log(rawDisclosures);
```

Comment:
Is this console.log wanted?

Reply:
the example app is on /example/src instead this is on /src

Reply:
Oops 😳
Fixed!